cassandra-bridge/src/main/java/org/apache/cassandra/spark/utils/MapUtils.java

```diff
@@ -225,4 +225,15 @@ public static String getOrDefault(Map<String, String> options, String key, Strin
     {
         return options.getOrDefault(lowerCaseKey(key), defaultValue);
     }
+
+    /**
+     * Method to check if key is present in {@code options} map.
+     * @param options   the map
+     * @param key       the key to the map
+     * @return boolean
+     */
+    public static boolean contains(Map<String, String> options, String key)
```

Comment:
should this method be called `containsKey` instead?

Reply:
makes sense, will make it consistent with `Map`